testRunner.html

```diff
@@ -16,7 +16,13 @@
 // Unit tests
 var metadata = {traceurOptions: {sourceMaps: true}};
 System.import('./test/unit/unitTestRunner.js', metadata).then(function(module) {
-  return module.unitTestRunner.run().then(function() {
+  var unitTestRunner = module.unitTestRunner;
+  // TODO(jjb): remove when feature tests are es6
```

Comment:
This TODO cannot be fixed since feature tests test Script behavior which cannot contain import declarations.